server/server.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"log"
+	"net/http"
+	"server/api"
+	"time"
+
+	"github.com/gorilla/mux"
+)
+
+func main() {
+	r := mux.NewRouter()
+	r.Methods(http.MethodPost).Path("/api/order").HandlerFunc(api.CreateOrder)
```

Comment:
Should we use https instead?

Reply:
This will be finally deployed as Azure Web Service. Https is handled by Azure Web Service.